npm/ng-packs/packages/components/extensible/src/lib/components/abstract-actions/abstract-actions.component.ts

```diff
@@ -16,9 +16,8 @@ export abstract class AbstractActionsComponent<
 
   @Input() record!: InferredData<L>['record'];
 
-  constructor(injector: Injector) {
+  protected constructor(injector: Injector) {
```

Comment:
Why protected 🤔 it's already abstract class

Reply:
because it is abstract class. It only uses when inherited. I know it is pointless but I did it. }:)